packages/bento-web/src/pages/api/profile/index.ts

```diff
@@ -0,0 +1,51 @@
+import { PostgrestError, PostgrestResponse } from '@supabase/supabase-js';
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+import { UserProfile } from '@/profile/types/UserProfile';
+import { Supabase } from '@/utils/Supabase';
+
+type APIRequest = NextApiRequest & {
+  body: UserProfile;
+};
+
+export default async (req: APIRequest, res: NextApiResponse) => {
+  const { ...Profile } = req.body;
```

Comment:
```suggestion
  const profile = req.body;
```
1. Profile -> profile
2. `{ ...profile } =` 대신 `profile =`

Reply:
`const { body: profile } = req;` 로 변경했습니다.